_posts/release/2020-01-19-accumulo-2.1.0.md

```diff
@@ -1,10 +1,20 @@
 ---
 title: Apache Accumulo 2.1.0
+sortableversion: '02.01.00'
+archived_critical: true
```

Comment:
```suggestion
sortableversion: '02.01.00'
```

Archiving it for critical bugs is a bit premature, given that it hasn't been released yet. :smiley_cat: